shared/src/lib/utils.js

```diff
@@ -13,7 +13,9 @@ export async function summarizeContent({
 }) {
   let summary = 'Unknown error';
   let success = false;
-  const useApi = Boolean(api_token);
+  const useApi = Boolean(
+    api_token && ((api_engine && api_engine !== 'cecil') || text),
+  );
```

Comment:
Now we'll only use the API if there's a an API Key _and_ the engine isn't Cecil.